src/components/Wiki/History/NoHistoryView.tsx

```diff
@@ -0,0 +1,44 @@
+import { LinkButton } from '@/components/Elements'
+import { Flex, Box, Image, Text } from '@chakra-ui/react'
+import React from 'react'
+import { Wiki } from '@/types/Wiki'
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
+      direction={{ base: 'column' }}
+      mt={{ base: '22' }}
+      mb={{ base: '96' }}
```

Comment:
```suggestion
      mt={{ base: '12', lg: '24' }}
```
margin bottom is giving too much white space the bottom. i think the page padding is enough for balance. 22 is not in chakra spacing list so its applying 22px. make it 24 so it gives enough space. also added 12 to make the space smaller in mobile devices and to be consistent with margin top on the sentence